resqpy/organize/__init__.py

```diff
@@ -2,5 +2,24 @@
     'OrganizationFeature',
```

Comment:
Please add all of the imports below into this list, as it helps with the documentation script.

Reply:
Done